benches/benchmark.rs

```diff
@@ -85,6 +85,7 @@ fn generate_crl(revoked_count: usize) -> Vec<u8> {
         crl_number: SerialNumber::from(1234),
         alg: &PKCS_ECDSA_P256_SHA256,
         key_identifier_method: KeyIdMethod::Sha256,
+        issuing_distribution_point: None,
```

Comment:
I'm confused by this. This field can't have been added in a semver-compatible rcgen bump, right?

Reply:
The entire `CertificateRevocationListParams` struct (with this `issuing_distribution_point` field) was new in 0.11.2. Since the struct was brand new, it was a semver compatible change.

It _appears_ to be a breaking change here because the `benchmark.rs` code was written when we had pinned `rcgen` to a commit in `main` between the point where `CertificateRevocationListParams` was added, and the `issuing_distribution_point` field being added.

Does that make sense?

Reply:
Ahh, yes!